### PR TITLE
remove bmp-js dependency

### LINKED ISSUE #319: bmp tests might be incorrect
In the test suite BMP images are being decoded using [bmp-js](https://github.com/shaozilee/bmp-js) which decodes into ABGR.
All other formats are being decoded by [node-canvas](https://github.com/Automattic/node-canvas) which decodes into RGBA.

As far as I can tell, there is no conversion from ABGR => RGBA happening for bmp's but tests are still passing.
If I force the use of canvas to decode bmp images then the tests start failing.

I believe this means the bmp tests are incorrect, right? Sorry if I'm missing something here.

Thanks!

### PR DESCRIPTION
**Pull request by jkeys089**
This PR removes bmp-js and relies purely on node-canvas for image decoding.
As discussed in https://github.com/obartra/ssim/issues/319, bmp-js decodes images as ABGR, but we need RGBA.

Note: a small change to the CI setup was needed to get node-canvas installed/running on a (relatively) modern version of node.

Closes  https://github.com/obartra/ssim/issues/319

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/repo
     docker:
-      - image: circleci/node:latest
+      - image: cimg/node:16.17.0
```
Comment (obartra) [inline .circleci/config.yml:6]: does it break with latest node?

**Review by obartra — APPROVED** [review]
Looks good! Thanks for the contribution! 🎉 

Left a question, nothing blocking just curious. If we can have it in latest (or a more recent version) without further changes I would favor it. Otherwise let me know and I can merge as is

File: .circleci/config.yml (hunk 1 above)
Comment (jkeys089) [reply .circleci/config.yml:6]: @obartra yes -- I believe the older version of node-canvas is incompatible with more modern versions of node. We could upgrade the node-canvas dependency but I guess that should be a separate PR.

Comment (obartra) [reply .circleci/config.yml:6]: absolutely 👍

**Pull request merged**